bindings/pyroot_experimental/cppyy/CPyCppyy/src/CPyCppyyModule.cxx

```diff
@@ -431,7 +431,7 @@ PyObject* AsCObject(PyObject* dummy, PyObject* args)
 // Return object proxy as an opaque CObject.
     void* addr = GetCPPInstanceAddress(dummy, args);
     if (addr)
-        return CPyCppyy_PyCapsule_New((void*)(*(intptr_t*)addr), nullptr, nullptr);
+        return CPyCppyy_PyCapsule_New((void*)addr, nullptr, nullptr);
```

Comment:
I am lost in this simplification, but it looks good. Do you know why before it was going through an int ptr(ptr)?

Reply:
It was just a mistake. There was an extra dereference that should not be there in the first place, since we get the object address already from `GetCPPInstanceAddress`. I pointed this out to Wim in:
https://bitbucket.org/wlav/cppyy/issues/100/ascobject-passes-wrong-address-to
and he committed the same change to upstream Cppyy:
https://bitbucket.org/wlav/cpycppyy/commits/dd55fb891de9a99a5094a25dbf05830da53ad6e0